src/store/participantsStore.js

```diff
@@ -85,16 +85,14 @@ const getters = {
 	 * Gets the array of external session ids.
 	 *
 	 * @param {object} state - the state object.
-	 * @param {object} rootState - the root state object.
 	 * @return {Array} the typing session IDs array.
 	 */
-	externalTypingSignals: (state, rootState) => (token) => {
-		if (state.typing[token]) {
-			return Object.values(state.typing[token]).filter(id => {
-				return rootState.getSessionId() !== id
-			})
+	externalTypingSignals: (state) => (token) => {
+		if (!state.typing[token]) {
+			return []
```

Comment:
Isn't it an object now, not an array?

Reply:
state.typing[token] is object: {
  [key: sessionId] : { expirationTimeout }
}

getter returns an array of other participants sessionIds (or empty array), because we don't need to expose expirationTimeout somewhere